spec/codecs/compress_spooler_spec.rb

```diff
@@ -143,6 +143,33 @@
           expect(buffer.size).to eq(2)
         end
       end
+
+      context "message spooling if min flush time is set" do
+        let(:spool_size) { 10 }
+        min_flush_time = 2
```

Comment:
can we make this var also inside a let? is discouraged to have this kind of statements in rspec.